build.xml

```diff
@@ -184,7 +184,7 @@
     <property name="opendiabetes.output.dir"
               value="${module.opendiabetes.basedir}/out/production/OpenDiabetes"/>
     <property name="opendiabetes.testoutput.dir"
-              value="${module.opendiabetes.basedir}/out/test"/>
+              value="${module.opendiabetes.basedir}/out/test/OpenDiabetes"/>
```

Comment:
Why changing the output directory here? shouldn't it be out/test/

Reply:
Yes it should, seems I tried something and forgot to revert this.

Reply:
Done by cabcb05